docs/api/sql/Function.md

```diff
@@ -1082,6 +1082,29 @@ SELECT ST_IsClosed(ST_GeomFromText('LINESTRING(0 0, 1 1, 1 0)'))
 
 Output: `false`
 
+## ST_IsCollection
+
+Introduction: Returns `TRUE` if the argument type is a geometry collection type.
```

Comment:
The doc here is not comprehensive. Returns TRUE if the geometry type of the input is a geometry collection type. Collection types are the following:

GEOMETRYCOLLECTION

MULTI{POINT,POLYGON,LINESTRING}